### ISSUE TITLE
Incorrect links

### ISSUE THREAD

**Issue by DazWilkin** [Nakazoto/CenturionComputer#4]
All the [Links](https://github.com/Nakazoto/CenturionComputer/wiki#links) at the end of the wiki [Home](https://github.com/Nakazoto/CenturionComputer/wiki) are broken (404).

E.g. `Emulators` links to https://github.com/Nakazoto/CenturionComputer/Emulators-and-Simulations

The sidebar is correct and contains the correct links. In the above case:

https://github.com/Nakazoto/CenturionComputer/wiki/Emulators-and-Simulations

Am unsure how to edit the Wiki Markdown as I don't use GitHub Wiki:

Currently: 
```
[Emulators](../Emulators-and-Simulations)
```
Perhaps?
```
[Emulators and Simulations](/Emulators-and-Simulations)
```

**Comment by Nakazoto** [comment]
Yeah, someone recently "fixed" the links on that page and somehow managed
to break them.
Thanks for lettting me know, I'll fix the links again.
